CONTRIBUTING.md

```diff
@@ -198,22 +198,29 @@ Add them for all new code, with the exception of:
 Make sure you import the types from `net.jcip`, there are homonyms in the classpath.
 
 
+### Nullability annotations
+
+We use the [Spotbugs annotations](https://spotbugs.github.io) to document nullability of parameters,
+method return types and class members.
+
+Please annotate any new class or interface with the appropriate annotations. Make sure you import 
+the types from `edu.umd.cs.findbugs.annotations`, there are homonyms in the classpath.
+
+
 ## Coding style -- test code
 
 Static imports are permitted in a couple of places:
-* AssertJ's `assertThat` / `fail`.
-* Some Mockito methods, provided that you're already using a non-statically imported method at the
-  beginning of the line. For example:
+* AssertJ methods whose names start with `assertThat` or `fail`.
```

Comment:
Adding this to also allow static import of `assertThatThrownBy`. There is also `catchThrowable` which is very convenient, not sure if we should include it or not.

Reply:
At this point we might as well say "all AssertJ methods". They're pretty recognizable since you know you're looking at test code.